### DISCUSSION TITLE
v2/components/zh/guide/bilibili

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#398] [Announcements]
# v2/components/zh/guide/bilibili

在 Markdown 文件中嵌入 B 站视频。 案例 一个 B 站视频: 一个自定义分 P 和空降地址的 B 站视频: 一个自定义设置的 B 站视频: 属性 bvid 类型: string; 必填: 是; B 站视频 bvid。 page 类型: number; 默认值: 1; 视频分 P。 width 类型: string | number; 默认值...

https://vuepress-theme-hope.github.io/v2/components/zh/guide/bilibili.html

**Comment by zzxzzk115** [comment]
不知道 Mr.Hope 是否和我一样觉得 BiliBili 组件在指定 time 参数后，应该在那个时间点暂停住，而不是自动播放。

**Reply by Mister-Hope** [reply]
B 站 iframe 没有这类参数，传 time 的默认行为就是从那个时候开始播放
